seqr/views/apis/dataset_api.py

```diff
@@ -110,6 +112,37 @@ def add_variants_dataset_handler(request, project_guid):
     Family.bulk_update(
         request.user, {'analysis_status': Family.ANALYSIS_STATUS_ANALYSIS_IN_PROGRESS}, guid__in=family_guids_to_update)
 
+    if project.workspace_name and project.workspace_namespace:
```

Comment:
So as I've said before, this flow does not match the one specified in the ticket. If it is as an internal project, REGARDLESS OF WHETHER IT IS IN ANVIL OR NOT, it needs to send the notification

Reply:
I see. I need to switch around the conditions for internal and AnVIL projects.